### ISSUE TITLE
Fallen Meteors: Buggs Around When Mined

### ISSUE THREAD

**Issue by AntonioNoack** [micdoodle8/Galacticraft#2365]
When you mine a fallen meteor it will need much more time to disappear than any other block.

Also you might know the sticky resin from IC2: it's the same behavior of the item which is dropped by the fallen meteor: it buggs around, flies like 10m away and then gets a reset from the server and is teleported in another direction.

It would be nice if you could fix to so that fallen meteors are mineable like all other blocks,

**Comment by dj3520** [comment]
is this 1.7.10 or 1.8.9?

**Comment by AntonioNoack** [comment]
1.7.10

**Comment by dj3520** [comment]
Believe this is an issue in vanilla that is fixed in newer versions.

**Comment by radfast** [comment]
I'm not seeing this issue in testing.

When you mine a block, the item dropped in always spawned **server-side** in Minecraft.  If the server-client connection is laggy (high ping) or the server has other problems then item drops will appear after a delay, the infamous "block lag".

The **Fallen Meteor** makes its drops the same way as any other block which drops items.

The only difference is that the Fallen Meteor is a tough block to mine - as tough as Obsidian!

I'll adjust that now, that's a bit **too** tough.  At the same time, I've made it mine faster if you use proper tools:
* Diamond Pickaxe: as now, but I've made the block overall 20% easier to mine
* Galacticraft Steel Pickaxe: mines twice as quickly
* Desh Pickaxe: same, twice as quickly (Desh items are not speedy but have very high durability)
* Titanium Pickaxe: mines three times as quickly

Equivalent power level tools from other mods - for example high end Tinker's Construct tools - will be similar.
